README.md

```diff
@@ -1,2 +1,43 @@
 # search-services
-Centralized configuration for services used by Search.gov applications
+This repo includes a centralized configuration for services used by Search.gov applications, including:
+
+- [search-gov](https://github.com/GSA/search-gov)
+- [i14y](https://github.com/GSA/i14y)
+- [ASIS](https://github.com/GSA/asis)
+
+## Prerequisites
+In order to run the services, you will need to install [Docker](https://www.docker.com/get-started).  We recommend setting the max memory alloted to Docker to 4GB (in Docker Desktop, Preferences > Resources > Advanced). See [the wiki](https://github.com/GSA/search-services/wiki/Docker-Command-Reference) for more documentation on basic Docker commands.
+
+## Services
+The docker-compose.yml file configures each service. You can run all the services with a single command:
+```
+docker compose up
+```
+Alternatively, you can run a subset of the services, i.e.:
+```
+docker compose up mysql elasticsearch7
+```
+
+Each repo uses one or more of the following services:
+
+### [MySQL](https://dev.mysql.com/doc/refman/5.7/en/)
+Database backend required by search-gov.
+
+### [Elasticsearch](https://www.elastic.co/elasticsearch/)
+All of our applications use Elasticsearch 6 in production for full-text search. Locally, it runs on the default port [9200](http://localhost:9200/). **Elasticsearch 6 is not supported on M1 Macs.** Until all our repos use Elasticsearch 7 in mid-2022, Developers on M1s can run the remaining services:
+```
+docker compose up mysql elasticsearch7 redis tika kibana7
+```
+You will need to update the Elasticsearch host in each repo's `config/secrets.yml` or `config/elasticsearch.yml` file to specify `localhost:9207`.
```

Comment:
@MothOnMars Can this be wordsmithed/rearranged a little?   On first cursory glance it was unclear to me that "You will need to update the Elasticsearch host in each repo's `config/secrets.yml` or `config/elasticsearch.yml` file to specify `localhost:9207`" only applies to those running ES7 because of the documented M1 issue.  I could see how this could be confusing to new developers who might assume you ALWAYS need to update those configs, since that guidance is given under the main Elasticsearch heading.  I advise adding subheadings for ES6 and ES7 under an ES main heading; adding a line break before mentioning the M1 setup under the ES6 sub-subheading; and rewording some of that M1 info to make it clear it only applies to this exception, not a general rule.

So, something like:

```
### [Elasticsearch](https://www.elastic.co/elasticsearch/)
Services for Elasticsearch 6 and 7 are both provided here.

#### Elasticsearch 6
All of our applications use Elasticsearch 6 in production for full-text search. Locally, it runs on the default port [9200](http://localhost:9200/). 

**Please note:  Elasticsearch 6 is not supported on M1 Macs.** Until all our repos use Elasticsearch 7 (estimated mid-2022), Developers on M1s can run the remaining services with:
```
docker compose up mysql elasticsearch7 redis tika kibana7
```
then update the Elasticsearch host in each repo's `config/secrets.yml` or `config/elasticsearch.yml` file to specify `localhost:9207` to point to the Elasticsearch 7 instance.

#### Elasticsearch 7
Currently only used in development, running on port [9207](http://localhost:9207/).
```